### PR TITLE
test: fix device attribute provider test cases

### PR DESCRIPTION
**Pull request by ami-aman**

### CODE REVIEW

**Review by Shahroz16 — APPROVED** [review]
Looks good, since we are not updating device attributes differently, is that going to be a separate PR?

File: Tests/Tracking/Util/DeviceAttributesProviderTest.swift [hunk 1]
```diff
@@ -55,23 +56,6 @@ class DeviceAttributesProviderTest: UnitTest {
 
         waitForExpectations()
     }
-
-    func test_getSdkVersionAttribute_givenNotUsingSdkWrapper_expectGetSDKVersion() {
```
Comment (mrehan27) [inline Tests/Tracking/Util/DeviceAttributesProviderTest.swift:59]: Shouldn't we still have test to verify correct SDK version being attached? Or do we plan add more tests for `DeviceAttributesProvider` in later PR?

Comment (ami-aman) [reply Tests/Tracking/Util/DeviceAttributesProviderTest.swift:59]: I would add the test cases in another PR. This PR is just a fix to the broken test cases that was being a blocker to run other tests cases.

**Comment by ami-aman** [comment]
@Shahroz16 
> since we are not updating device attributes differently, is that going to be a separate PR
Apologies but I think I am not sure what exactly you are referring to, to be done in a separate PR? 🤔

**Comment by Shahroz16** [comment]
> @Shahroz16
> 
> > since we are ~~not~~ now updating device attributes differently, is that going to be a separate PR
> 
> Apologies but I think I am not sure what exactly you are referring to, to be done in a separate PR? 🤔

I meant do you plan to add tests for Device attributes? since now we relying on Datapipeline for device attributes, but i talked offline with Rehan about it, we will setup a framework for it first.

**Pull request merged**